src/model/handlers.py

```diff
@@ -88,12 +92,14 @@
 
 
 async def model_get(request):
+    logger.debug("Request handler called: model_get")
```

Comment:
These log messages cannot be made visible by default in development mode (I assume that `logger.debug` is only meant for development mode). This looks like plenty of duplication to me, especially because it could be automated easily with a decorator (I don't know how is it different to decorate an async function, but `Request handler called: ` is the same, the function name is available as `__name__`

Reply:
> These log messages cannot be made visible by default in development mode

What do you mean?

Yes, it's intended for development mode (but we might enable `debug` level output in production if it's useful and not too noisy)

Reply:
That's exactly what I meant. All these functions are request handlers, I thought there should be a better / automatic way of logging on the console when an endpoint was reached in development mode. Perhaps a middleware could do this?

Reply:
@matyasfodor I've refactored to use a middleware. I don't see a big pragmatical difference, but I'm happy as long as the logs get out there one way or the other.